spec/spec_helper.rb

```diff
@@ -2,8 +2,10 @@
 Bundler.setup(:test)
 require 'codeclimate-test-reporter'
 CodeClimate::TestReporter.start
+require 'capybara/poltergeist'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.